lms/djangoapps/edxnotes/tests.py

```diff
@@ -1077,7 +1077,12 @@ def test_get_id_token(self):
         response = self.client.get(self.get_token_url)
         assert response.status_code == 200
         client = Application.objects.get(name='edx-notes')
-        jwt.decode(response.content, client.client_secret, audience=client.client_id)
+        jwt.decode(
+            response.content,
+            client.client_secret,
+            audience=client.client_id,
+            algorithms=[settings.JWT_AUTH['JWT_ALGORITHM']]
```

Comment:
I've only made the latest commit. the previous commits are from this PR: https://github.com/edx/edx-platform/pull/28475

Since I'm upgrading edx-drf-extensions and that brings with it pyjwt2 I need to base on that PR since it adds support for that new library. 

I'm going to hold of on more work on this till that PR is merged since it's foundational to this work.